### PR TITLE
Add total breakdown to All publishers report

### PR DESCRIPTION
**Pull request by ericholscher**
This will enable us to more easily compare Day over Day metrics across a subset of our stack.

### CODE REVIEW

File: adserver/templates/adserver/reports/all-publishers.html [hunk 1]
```diff
@@ -40,12 +40,14 @@
 {% block toc %}
 <section class="mb-5">
   {% if publishers %}
-    <h2>{% trans 'Publishers '%}</h2>
+    <h2>{% trans 'Publishers '%} ({{ publishers|length }})</h2>
+    <details>
```
Comment (davidfischer) [inline adserver/templates/adserver/reports/all-publishers.html:44]: I assume you're aware but this will hide these by default. I think I'm OK with that.

**Review by davidfischer — APPROVED** [review]
This is great and I wanted this again today!

File: adserver/templates/adserver/reports/all-publishers.html (hunk 1 above)
Comment (ericholscher) [reply adserver/templates/adserver/reports/all-publishers.html:44]: Yea -- I don't think its super valuable to take up the first 2 pages of screen, and easy enough to open if needed.

Comment (ericholscher) [reply adserver/templates/adserver/reports/all-publishers.html:44]: @davidfischer I actually want a way to hide the per-publisher breakdown table as well, but `details` doesn't work on tables, so I punted for now :)

**Pull request merged**